auth-service/src/main/kotlin/uk/gov/communities/delta/auth/services/GroupService.kt

```diff
@@ -45,19 +55,50 @@ class GroupService(
         groupCN: String,
         call: ApplicationCall,
         triggeringAdminSession: OAuthSession? = null,
+    ) {
+        addUserToGroup(adUser.cn, adUser.dn, groupCN, call, triggeringAdminSession)
+    }
+
+    suspend fun addUserToGroup(
+        userCN: String,
+        userDN: String,
+        groupCN: String,
+        call: ApplicationCall,
+        triggeringAdminSession: OAuthSession? = null,
```

Comment:
I think I'd rather this wasn't optional, makes it harder to leave it out by accident